Readme.md

```diff
@@ -7,7 +7,7 @@ Web-based service for building games with LÖVE Potion
 1. Make sure you are running Python >= 3.11
 2. Clone this repository with `git clone --recurse-submodule https://github.com/lovebrew/lovebrew-webserver.git`
 3. Run `pip install .` inside the repository directory
-4. alternatively you can use venv to avoid conflic with your local packages see https://docs.python.org/3/library/venv.html for more details
+4. alternatively you can use venv to avoid conflicts with your local packages see https://docs.python.org/3/library/venv.html for more details
```

Comment:
Small nitpicks:

1. Capitalize 'alternatively' to 'Alternatively'
2. Use Markdown links, e.g: "Alternatively, you can use [venv to avoid local package conflicts](https://docs.python.org/3/library/venv.html)"